packages/wdio-cli/tests/utils.test.js

```diff
@@ -248,3 +250,32 @@ describe('addServiceDeps', () => {
         global.console.log.mockClear()
     })
 })
+
+describe('getCapabilities', () => {
+    it('should return driver with capabilities for android', () => {
+        expect(getCapabilities({ option: 'foo.apk' })).toEqual({ ...ANDROID_CONFIG, app: 'foo.apk' })
+        expect(getCapabilities({ option: 'android' })).toEqual({
+            ...ANDROID_CONFIG,
+            browserName: 'Chrome'
+        })
+    })
+
+    it('should return driver with capabilities for ios', () => {
+        expect(getCapabilities({ option: 'foo.app', device: 'fooName', udid: 'num', ver: 'fooNum' })).toEqual({
+            ...IOS_CONFIG,
+            app: 'foo.app',
+            udid: 'num',
+            deviceName: 'fooName',
+            platformVersion: 'fooNum'
+        })
+        expect(getCapabilities({ option: 'ios' })).toEqual({
+            ...IOS_CONFIG,
+            browserName: 'Safari',
+            platformVersion: null,
+            udid: null })
+    })
+
+    it('should return driver with capabilities for desktop', () => {
+        expect(getCapabilities({ option: 'chrome' })).toEqual({ browserName: 'chrome' })
+    })
+})
```

Comment:
The object checks could be snapshots instead.

For example
```js
expect(getCapabilities({ option: 'chrome' })).toMatchSnapshot()
```

This way we don't have to worry about changing the objects in the tests if we change something in the function's return. We just update the snapshots.